chronos/chronos_plugin.py

```diff
@@ -94,33 +94,33 @@ def __init__(self, params):
             alarm_constants.CHRONOS_NOT_YET_CLUSTERED)
         pdlogs.NOT_YET_CLUSTERED_ALARM.log(cluster_desc=self.cluster_description())
 
-    def key(self): # pragma: no cover
+    def key(self):  # pragma: no cover
```

Comment:
Does the space change mean anything, or are you just making these all consistent?